support/cas-server-support-duo-core-mfa/src/main/java/org/apereo/cas/adaptors/duo/authn/DuoCredential.java

```diff
@@ -23,20 +24,33 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @EqualsAndHashCode(of = {"username"})
-public class DuoCredential implements Credential {
+public class DuoCredential implements Credential, MfaCredential {
```

Comment:
As noted early, this needs to be implemented by every form of MFA credential and not just Duo.